docs/usage.md

```diff
@@ -421,6 +421,19 @@ Currently there are not yet any zarr v3 readers which understand the chunk manif
 This store can however be read by {py:func}`~virtualizarr.xarray.open_virtual_dataset`, by passing `filetype="zarr_v3"`.
 ```
 
+## Opening Kerchunk references from disk as virtual datasets
+
+You can open `json` or `paruqet` Kerchunk references from disk as virtual datasets. This may be useful in appending workflows or creating checkpoints for larger datasets.
```

Comment:
Or converting kerchunk references into different references formats.

Reply:
I'll update and fix my spelling of ~~paruqet~~ parquet